### DISCUSSION TITLE
Label Question: Directive ICE

### DISCUSSION THREAD

**Discussion by ianknowland** [CommonCoreOntology/CommonCoreOntologies#577] [Ontology Terms and Class Schema]
Pretty simple:

- Designative ICE --> designates
- Descriptive ICE --> describes
- Representative ICE --> represents

but

- Directive ICE --> prescribes

Why is this not a Prescriptive ICE, if the _is about_ subproperties are meant to correlate with their respective ICE? I did not find another discussion or issue on this, so maybe it's not much of an issue as everyone comfortably uses the given terminology. But at a glance, I don't see a good reason why Directive fits better than Prescriptive, given that a possible source definition of the latter is "relating to the imposition or enforcement of a rule or method". Seems like an easy lateral move and maintains consistency.

In the end, I am merely irked by the misalignment :)

**Comment by gregfowlerphd** [comment]
As a fan of consistency, I feel favorably towards this suggestion.

**Comment by michaelrabenberg** [comment]
Isn't your bullet 1 incorrect? Here's the designative ICE def: 

An Information Content Entity that consists of a set of symbols that _denote_ some Entity.

So in fact the situation is even worse than you indicated--"total anarchy" seems like an apt characterization.

I'd approve of eliminating these misalignments. In both cases (the designative one and the directive one) the question is whether to change the def to match the label or to change the label to match the def. The labels enjoy a lot of established usage (and other things are defined in terms of them) so changing the defs would presumably be a less invasive way to go.

**Reply by ianknowland** [reply]
Thank you, I didn't catch that! I was looking at the object properties directly. That one at least seems to be an oversight for whoever wrote the definition, as "designates" has the proper domain of Designative ICE.

To your point, at least in the case of Directive ICE it seems to make more sense to change the label given that the alternative is to change "prescribes" to "directs" in the definition, which I think loses something important--namely, the useful duality of "descriptive" and "prescriptive".  Not to mention, changing the definition for Directive ICE to match the current label would essentially mean a change to an object property label, which could be a toss up as "prescribes" is widely used as well.

**Comment by gregfowlerphd** [comment]
Given your concern with consistency here, @ianknowland, it's worth noting another difference between the four classes: Whereas Designative ICE, Directive ICE, and Representational ICE are each provided with an equivalence axiom defining them in terms of the corresponding object property, Descriptive ICE is not. This inconsistency should probably also be remedied.
